### DISCUSSION TITLE
Failed to Synchronise information with the cloud

### DISCUSSION THREAD

**Discussion by gi921120** [opengisch/QField#2533] [Q&A]
Hi everyone, 

I'm getting this error when I tried to sync some information to the cloud using the plugin in qgis. I tried this with different vector layers and I got the same result.  

![image](https://user-images.githubusercontent.com/61034374/155272595-bf26ce4b-638f-4b50-8d1f-287e6b3b6bb5.png)

**Comment by juanotopo** [comment]
Hello, I have the same problem.

someboy have a solution?

**Comment by suricactus** [comment]
hello, @gi921120 ,

Looking at your screenshot, it says the request timed out. Did you wait a lot? Can you share a bit more details about your setup - QGIS, QFieldSync versions. Have you tried in a new QGIS profile?

@juanotopo Can you share a bit of your setup too.

We are unable to reproduce this on multiple machines, but obviously there is a trouble and we need your help to find out what it is.

**Reply by gi921120** [reply]
Hi Ivan@suricactus,

I didn't wait at all, I received the message inmidiately when I tried syncronizing some information. I tried creating and empty the project in the webview and then I opened it and sync the information with the plugin. Similarly, I tried creating a simple project with some vector information with "Convert currently open project to cloud project" and received the same message. Please see picture below. 

Please let me know if you want me to send you screenshots of the whole routine. 

![image](https://user-images.githubusercontent.com/61034374/156864633-fc3a8166-54de-4ea4-9188-d5c4270ff7d1.png)

**Reply by suricactus** [reply]
Thanks! Your input helped to find out and solve the issue. It was specific to 3.16 and it is going to be fixed in the upcoming QFIeldSync BETA20.

**Comment by alansaviolobo** [comment]
![image](https://user-images.githubusercontent.com/1202289/156155922-05a9d334-a247-4076-a995-462073faa7b2.png)

I am facing a similar issue

qgis version 3.16.3-Hannover
qfield sync Installed version | 4.0.0-BETA17

**Comment by suricactus** [comment]
Hello, @gi921120  @alansaviolobo  @juanotopo . Thanks for reporting this issue. It was 3.16 specific. The fix is coming in BETA20 very soon. More about the issue is here https://github.com/opengisch/qfieldsync/pull/394
